README.md

```diff
@@ -58,8 +58,8 @@ To run one of them, use `./bin/run-example <class> [params]`. For example:
 will run the Pi example locally.
 
 You can set the MASTER environment variable when running examples to submit
-examples to a cluster. This can be a mesos:// or spark:// URL,
-"yarn-cluster" or "yarn-client" to run on YARN, and "local" to run
+examples to a cluster. This can be a mesos:// or spark:// URL, to run on YARN; either --master yarn and set --deploy-mode (cluster or client) or simply set --master as 
```

Comment:
The sentence doesn't quite work now; I think it has to be:

"... spark:// URL, "yarn" to run on YARN, or "local" ..." 

In this high-level README it's not necessary to explain the alternative again. None of the alternatives are explained here in full.